website/content/docs/commands/pki/health-check.mdx

```diff
@@ -234,7 +259,13 @@ This health check checks each role to see whether or not it allows wildcard issu
 
  - `allowed_roles` `(list: nil)` - an allow-list of roles to ignore.
 
-This health check checks each role to see whether it sets `no_store=false` or not. When used with lots of certificates and no temporal auto-rebuilding of CRLs, this can result in bad performance. Instead, BYOC can be used for the occasional revocation and shorter certificate lifetimes used instead. Without auto-rebuild, this will be a warning, but when auto-rebuild is enabled, we can make it informational.
+This health check checks each role to see whether it sets `no_store=false` or not. When used with lots of certificates and no temporal auto-rebuilding of CRLs, this can result in bad performance. Instead, [BYOC](/vault/api-docs/secret/pki#revoke-certificate) can be used for the occasional revocation and shorter certificate lifetimes used instead. Without auto-rebuild, this will be a warning, but when auto-rebuild is enabled, we can make it informational.
+
+*NOTE* if the role is used for ACME issuance, `no_store=true` is required.
```

Comment:
```suggestion
```

Suggest deleting the stand-alone note and adding it to the remediation steps instead to make sure folks see it.